README.md

```diff
@@ -8,6 +8,14 @@
 
 Monorepo for the `ngworker` NPM organization. Packages for Angular applications.
 
+## Packages
+
+Packages in this monorepo.
+
+| Package                                                     | Description                                                                         |
+| ----------------------------------------------------------- | ----------------------------------------------------------------------------------- |
+| [Spectacular (@ngworker/spectacular)](packages/spectacular) | Spectacular Angular testing. Test harnesses for Angular applications and libraries. |
```

Comment:
Maybe a link to the Spectacular README?

Reply:
There's already a link to the Spectacular readme:
```
(packages/spectacular)
```